cli/crates/cli/tests/introspection_configuration.rs

```diff
@@ -0,0 +1,72 @@
+#![allow(unused_crate_dependencies)]
+mod utils;
+
+use backend::project::GraphType;
+use serde_json::Value;
+use utils::environment::Environment;
+
+use crate::utils::consts::INTROSPECTION_QUERY;
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+async fn introspection_configuration() {
+    let mut env = Environment::init();
+
+    env.grafbase_init(GraphType::Single);
+
+    env.write_schema_without_introspection(
+        r#"
+        extend type Query {
+            hello: String! @resolver(name: "hello")
+        }
+        "#,
+    );
+    env.write_resolver("hello.js", "export default function Resolver() { return 'hello'; }");
+
+    env.grafbase_dev_watch();
+
+    let mut client = env.create_client().with_api_key();
+
+    client.poll_endpoint(30, 300).await;
+
+    let response = client.gql::<Value>(INTROSPECTION_QUERY).send().await;
```

Comment:
Yes, I plan to expand that test but as we need this to be released today haven't here, will do!

Reply:
It's only to point out that snapshotting like this is just a lot easier for GraphQL responses, wouldn't have taken you more time either. But don't get me wrong, not a critic. Somehow it's only that after `integration-tests`, that I realized we complicated our life so much with `dot_get!()` stuff rather than comparing against a snapshot, but well 🤷

Reply:
Yeah I understand, I meant that I was thinking of expanding it with more `dot_get!()` to check that specific scenario but decided to release as is for the moment and add it after (with a manual check that that is in fact the correct error). Will do it with a snapshot once I modify it